control-plane/pkg/reconciler/channel/channel.go

```diff
@@ -447,6 +448,7 @@ func (r *Reconciler) getSubscriberConfig(ctx context.Context, channel *messaging
 	egress := &contract.Egress{
 		Destination:   subscriber.SubscriberURI.String(),
 		ConsumerGroup: consumerGroup(channel, subscriber),
+		DeliveryOrder: contract.DeliveryOrder_ORDERED,
```

Comment:
perhaps.... not sure it really makes sense, but:

https://github.com/knative-sandbox/eventing-kafka-broker/blob/d3815d268fb6dd3d2a39c3043ea8606232a2165a/control-plane/pkg/reconciler/source/source.go#L50-L52


is that a change from the other `KafkaChannel`s?

Reply:
I'm not following what do you mean here.

Reply:
I also am not sure about the behaviour of delivery order in KafkaChannel implementations.

Reply:
is the `eventing-kafka` unordered ?

Reply:
eventing-kafka is ordered (_only_)